charms/knative-eventing/requirements.in

```diff
@@ -1,2 +1,2 @@
-ops
+ops>2.0
```

Comment:
Did you have to "force" it to pick greater versions than 2.0?

Reply:
current code will be incompatible with lower, right ? :)

Reply:
What I mean is, why did you have to "pin" `ops>2.0` instead of leaving it as just `ops` and let pip-compile fetch the latest version, which is 2.x.

Reply:
we already had issues when pip compile downgraded the package

if I know in advance that we support only ops>2
then why not to pin it?

if it will be compatible with both, then of course I will leave it as is

Reply:
there is no guarantee that pip compile goes into higher versions

thus, if you know that you need higher version, you do soft link, eg `>x.x.x`